waku/v2/node/peer_manager/waku_peer_store.nim

```diff
@@ -16,7 +16,9 @@ type
     # CanConnect: was recently connected to peer and disconnected gracefully.
     CanConnect,
     # Connected: actively connected to peer.
-    Connected
+    Connected,
+    # ShouldNotConnect: connection was refused
+    ShouldNotConnect
```

Comment:
If we are adding states to peer manager this should be updated in the https://rfc.vac.dev/spec/27/ spec. Is there any issue/PR in rfc repo for this?